monitoring/prometheus/alerts/ceph_default_alerts.yml

```diff
@@ -0,0 +1,154 @@
+groups:
+  - name: cluster health
+    rules:
+      - alert: health error
+        expr: ceph_health_status == 2
+        for: 5m
+        labels:
+          severity: critical
+          type: ceph_default
+        annotations:
+          description: Ceph in error state for more than 5m
+      - alert: unhealthy
+        expr: ceph_health_status != 0
```

Comment:
Right, the intention was to not have these alerts "interdependent" since I assumed people might pick and choose which of these alerts they want through silences. Also this can be grouped by the alertmanager.

Otoh I don't have terribly strong feelings about this, would also be up for changing unhealthy to just a health_warn alert.

Reply:
Yeah, that is where I was heading with this -- the health_warn would be better in this case IMO.